### DISCUSSION TITLE
Add To Major Linux Repos?

### DISCUSSION THREAD

**Discussion by arrowj** [supermerill/SuperSlicer#2123] [Q&A]
Any way SuperSlicer could be added to the repos for the major Linux distros? Cura and Prusa are both available in Fedora 35. Prusa 2.4 is there as a Flatpack. I installed Prusa and I can only imagine SuperSlicer would be even better, but the AppImage thing is kind of a drag. Heck I would even add a custom repo and install at the command line via dnf. I do not have the skills to do this myself, but if there is something I _can_ do to help facilitate it let me know.

**Comment by arrowj** [comment]
I bodged things together as follows:

1. Create a directory called **AppImage** inside **/home/~**.
2. Dump SuperSlicer's extracted directory into the **AppImage** directory.
3. Install [Pin It!](https://github.com/ryonakano/pinit) (available as a Flatpak from the Gnome Software store).
4. Create an entry using Pin It! for SuperSlicer giving it the path to the executable and an icon: /home/[user]/AppImages/SuperSlicer_2.3.57.8_linux64_211222/bin/superslicer and /home/[user]/AppImages/SuperSlicer_2.3.57.8_linux64_211222/resources/icons/SuperSlicer_192px.png respectively.

This adds SuperSlicer to the launcher with an appropriate icon, as well as being able to launch it via text search and add it to your favorites in the dash.

If releasing SuperSlicer as a Flatpak is not feasible perhaps the devs could add it to [AppImageHub](https://www.appimagehub.com)? In any case, while not elegant I think it is still worth the extra steps in order to use SuperSlicer rather than the alternatives.

**Reply by goferthat** [reply]
I was able to get this to work (using Pinit via flatpak), but it's clunky as heck .... 

Also, Arrowj's example was the only guide needed (and necessary). You have to be explicit with the exec. path ... Using pinit provided folder lookup utility DID NOT WORK for the exec or icon(png)

I won't be disappointed if I never need to use pinit, or un-managed appl images again. 

Dan
Ubuntu 18.04.6 LTS
